src/lib/data_source.py

```diff
@@ -276,6 +281,10 @@ def run(
         # Process each record to add missing cumsum or daily diffs
         data = infer_new_and_total(data)
 
+        if "backfill" in self.config and self.config["backfill"]:
```

Comment:
You can simplify this as `if self.config.get("backfill")`

Reply:
Replaced with a parse flag, and used .get()